s4e-backend/src/test/java/pl/cyfronet/s4e/TestDbHelper.java

```diff
@@ -15,8 +15,10 @@
     private final WMSOverlayRepository wmsOverlayRepository;
     private final PRGOverlayRepository prgOverlayRepository;
     private final SldStyleRepository sldStyleRepository;
+    private final InvitationRepository invitationRepository;
 
     public void clean() {
+        invitationRepository.deleteAll();
```

Comment:
are you sure if this line is necessary? i think that cascade from on institution should delete invitations :)